index.js

```diff
@@ -34,9 +38,10 @@ module.exports = function() {
 
       // Unfortunately rework unconditionally splits selectors by comma, so our
       // :matches(...) selector gets munged.
-      var actualSelectors = rule.selectors.join(',').replace(/:matches\(.*\)/g, function(substr) {
+      var actualSelectors = rule.selectors.join(',').replace(/:matches\(.*?\)/g, function(substr) {
```

Comment:
Sorry, not super familiar with what the combination of `*` and `?` is doing--is the `.` supposed to be both greedy and not?